### PR TITLE
V2 token command

### PR DESCRIPTION
**Pull request by lynnemorrison**
Add token issue command
Move to kube directory
Add unit tests

### CODE REVIEW

**Comment by lynnemorrison** [comment]
creating new PR with just token issue command

**Review by nluaces — COMMENTED** [review]
Minor thing while testing it:
```
Waiting for token status ...
Error: Token "" redeem not ready yet, check the logs for more information
```
To redeem a token I'm just passing the filename, not the name.

File: internal/cmd/skupper/token/kube/token_redeem.go [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+	KubeClient kubernetes.Interface
+	activeSite *v1alpha1.Site
+}
+
+func NewCmdTokenRedeem() *CmdTokenRedeem {
+
+	skupperCmd := CmdTokenRedeem{}
+
+	cmd := cobra.Command{
+		Use:     "redeem <name>",
```
Comment (nluaces) [inline internal/cmd/skupper/token/kube/token_redeem.go:46]: It should be:

`Use: "redeem <filename>"

File: internal/cmd/skupper/token/kube/token_redeem.go [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+	_, err = cmd.client.AccessTokens(cmd.namespace).Create(context.TODO(), &resource, metav1.CreateOptions{})
+	return err
+}
+
+func (cmd *CmdTokenRedeem) WaitUntilReady() error {
+
+	err := utils.NewSpinnerWithTimeout("Waiting for token status ...", int(cmd.flags.timeout.Seconds()), func() error {
+
+		token, err := cmd.client.AccessTokens(cmd.namespace).Get(context.TODO(), cmd.name, metav1.GetOptions{})
```
Comment (nluaces) [inline internal/cmd/skupper/token/kube/token_redeem.go:168]: `cmd.name` is always empty here

Comment (lynnemorrison) [reply internal/cmd/skupper/token/kube/token_redeem.go:169]: I must have removed that accidentally when cleaning up code.   Fixed

File: internal/cmd/skupper/token/kube/token_redeem.go (hunk 1 above)
Comment (lynnemorrison) [reply internal/cmd/skupper/token/kube/token_redeem.go]: fixed

**Comment by lynnemorrison** [comment]
> Minor thing while testing it:
> 
> ```
> Waiting for token status ...
> Error: Token "" redeem not ready yet, check the logs for more information
> ```
> 
> To redeem a token I'm just passing the filename, not the name.
kubect
failure is because of the name is not set in cmd.name.

**Review by nluaces — APPROVED** [review]

**Pull request merged**